devel/conventions/index.md

```diff
@@ -0,0 +1,20 @@
+---
+layout:   default
+class:    markdown
+---
+
+Conventions
+===========
+
+TaskCluster is a very open platform, and can easily descend into anarchy if we are not careful to enforce some standard approaches and conventions.
```

Comment:
How about: "TaskCluster is a very open platform! To help us with our day to day work, we are setting up a few standard approaches and conventions to help make administration and coordination between the humans using the platform easier."